### DISCUSSION TITLE
Is it possible to have TagTreePicker?

### DISCUSSION THREAD

**Discussion by whateverfw** [rsuite/rsuite#3081] [Q&A]
Is there an existing functionality of TagTreePicker, that looks like CheckTreePicker but with tags?
Or possibility of having nested tags that can be selected?

**Comment by simonguo** [comment]
Hi @whateverfw  Currently there is no such component as TagTreePicker, I think you can implement this feature through the `renderValue` method of CheckTreePicker.
https://rsuitejs.com/components/check-tree-picker/#custom-options
